documentation/tutorials/get-started.md

```diff
@@ -385,28 +390,37 @@ tickets =
       ticket
     end
   end
+```
 
+Find the tickets where the subject contains `"2"`. Note that the we're setting the ticket data that we're querying using `set_data`.
 
-# Show the tickets where the subject contains "2"
+```elixir
 Helpdesk.Support.Ticket
 |> Ash.Query.filter(contains(subject, "2"))
 |> Ash.DataLayer.Simple.set_data(tickets)
 |> Helpdesk.Support.read!()
+```
 
-# Show the tickets that are closed and their subject does not contain "4"
+Find the tickets that are _closed_ and their subject does _not_ contain `"4"`
+
+```elixir
 Helpdesk.Support.Ticket
 |> Ash.Query.filter(status == :closed and not(contains(subject, "4")))
 |> Ash.DataLayer.Simple.set_data(tickets)
 |> Helpdesk.Support.read!()
 ```
 
-The examples shown here could be implemented easily using things like `Enum.filter`, but the real power here is to allow you to use the same tools when working with any data layer. If you were using AshPostgres, the above code would be exactly the same, except for the call to `set_data/2`.
+The examples above could be easily implemented with `Enum.filter`, but the real power here is to allow you to use the same tools when working with any data layer. If you were using the (link?) AshPostgres data layer, the above code would be exactly the same, except we wouldn't need the call to `set_data/2`.
+
+> TODO talk about any practical use for Simple Data layer, or explain why it is like it is (so you have something presumably until you set a real data layer)
+
+`Ash.DataLayer.Simple` isn't actually very useful in practice so
```

Comment:
I'd disagree with this. Its plenty useful :)

Reply:
Need to describe how though at a high level, as per the comment above